common/source/docs/common-dronecan-servos.rst

```diff
@@ -0,0 +1,23 @@
+.. _common_dronecan_servos:
+
+===============
+DroneCAN Servos
+===============
+
+ArduPilot supports :ref:`DroneCAN<common-uavcan-setup-advanced>` servos. DroneCAN connected servos offer high noise immunity and longer lead runs that normal :ref:`PWM servos<comon-servo>`.
+
+-  `HiTec <https://hitecrcd.co.jp/industrial/uploads/SG33BL-CAN_2020-08-03.pdf>`__
```

Comment:
Thanks for this.  Here's a better link for the HiTec CAN servos https://www.hiteccs.com/actuators/products/search/servo/1/spec/1-18/accessory/0/sort/0

Reply:
done